Mathlib/GroupTheory/SpecificGroups/Cyclic.lean

```diff
@@ -564,6 +579,19 @@ theorem CommGroup.is_simple_iff_isCyclic_and_prime_card [Fintype α] [CommGroup
 #align comm_group.is_simple_iff_is_cyclic_and_prime_card CommGroup.is_simple_iff_isCyclic_and_prime_card
 #align add_comm_group.is_simple_iff_is_add_cyclic_and_prime_card AddCommGroup.is_simple_iff_isAddCyclic_and_prime_card
 
+section SpecificInstances
+
+instance : IsAddCyclic ℤ := ⟨1, fun n ↦ ⟨n, by simp only [smul_eq_mul, mul_one]⟩⟩
+
+instance (n : ℕ) : IsAddCyclic (ZMod n) :=
```

Comment:
Since instance name generation is current pretty bad, I'd be inclined to keep the old generated name